cmd/fyne/internal/commands/bundle.go

```diff
@@ -227,6 +215,32 @@ func (b *Bundler) doBundle(filepath string, out *os.File) {
 	writeResource(filepath, b.name, out)
 }
 
+func getOutputFile(filePath string, noheader bool) (file *os.File, close func() error, err error) {
+	fileModes := os.O_RDWR | os.O_CREATE | os.O_TRUNC
+	if noheader {
+		fileModes = os.O_RDWR | os.O_APPEND
+	}
+
+	f, err := os.OpenFile(filePath, fileModes, 0666)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			fyne.LogError("Unable to open output file", err)
+			return nil, nil, err
+		}
+
+		// try creating the file
+		f, err = os.Create(filePath)
+		if err != nil {
+			fyne.LogError("Unable to read, or create, output file : "+filePath, err)
+			return nil, nil, err
+		}
+	}
+
+	return f, func() error {
```

Comment:
surely `return f, f.Close, nil` is sufficient?

Reply:
yes it is. Got refactored and not simplified...